doc/dev_guide/release_management.rst

```diff
@@ -10,92 +10,166 @@ Release management
 
-4. **Stable**. Finally, after we see our beta versions run successfully in
-   production, usually in a few more months, during which we fix all incoming
-   bugs and add some minor features, we declare this MAJOR release series
-   stable.
+   -----------------|---------|---------|---------|------> (time)
+                      1/4 yr.   1/4 yr.   1/4 yr.
 
-Like Ubuntu, we distinguish two kinds of stable releases:
+*Support* means that we continue fixing bugs. We add bug fixes simultaneously
+into the following release series: LTS, last stable, beta, and alpha.
+If we look at the release diagram above, it means that the bug fixes are to be
+added into 1.10, 2.4, 2.5, and 2.6 release series.
 
-* **LTS (Long Term Support)** releases that are supported for 3 years
-  (community) and up to 5 years (paying customers). **LTS** release
-  is identified by MINOR version 10.
-* **Standard stable releases** are only supported a few months after the next
-  stable is out.
+To sum it up, once a quarter we release (?see the release diagram above for
+reference):
 
-"Support" means that we continue fixing bugs in a release.
+* next LTS release, e.g., 1.10.7
+* two stable releases, e.g., 2.3.3 and 2.4.2
+* beta version of the next release series, e.g., 2.5.1.
 
-We add commits simultaneously to three MAJOR releases:
+In all supported releases, when we find and fix an outstanding CVE/vulnerability,
+we release a patch for that but do not tag a new PATCH level version.
+Users will be informed about such critical patches via official Tarantool news
+channel (`tarantool_news <https://t.me/tarantool_news>`_).
 
-* **LTS** is a stable release which does not receive new features, and only gets
-  backward compatible fixes. Hence, following the rules of semver, LTS release
-  never has its MAJOR or MINOR version increased, and only gets PATCH level
-  releases.
+We also publish nightly builds, and use the fourth slot in the version
+identifier to designate the nightly build number.
 
-* **STABLE** is our current stable release, which may receive new features.
-  When the next STABLE version is published, MINOR version is incremented.
-  Between MINOR releases, we may have intermediate PATCH level releases as well,
-  which will contain only bug fixes. We maintain PATCH level releases for
-  two STABLE releases, the current and the previous one, to preserve support
-  continuity.
+.. _backward-incompatible:
 
-* **NEXT** is our next MAJOR release, and it follows the maturity
-  cycle described in the beginning. While NEXT release is in alpha state,
-  its MINOR is frozen at 0 and is only increased when the release reaches
-  BETA status. Once the NEXT release becomes STABLE, we switch the vehicle for
-  delivery of minor features, designating the previous stable release as LTS,
-  and releasing it with MINOR set to 10.
+.. note::
 
-To sum up, once a quarter we release:
+   A release series may contain backward incompatible changes in a sense that
```

Comment:
Sorry for the late feedback, but I would say 'may introduce' instead of 'may contain'. It would give better understanding where exactly such changes may appear.